packages/block-editor/src/checkbox-input/index.js

```diff
@@ -0,0 +1,49 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import { CheckboxInput } from '@crowdsignal/blocks';
+import { CheckboxInputIcon } from '@crowdsignal/icons';
+import attributes from './attributes';
+import EditFreeText from './edit';
+
+const settings = {
+	apiVersion: 1,
+	title: __( 'Single Checkbox', 'block-editor' ),
+	description: __(
+		'Do you need consent to something like terms of service?',
+		'block-editor'
+	),
+	category: 'crowdsignal-forms/form',
+	keywords: [
+		__( 'checkbox', 'block-editor' ),
+		__( 'terms of service', 'block-editor' ),
+	],
+	icon: <CheckboxInputIcon />,
+	edit: EditFreeText,
+	attributes,
+	supports: {
+		html: false,
+		reusable: false,
+		align: [ 'wide', 'full' ],
```

Comment:
I'm not sure if `wide` or `full` align makes sense for an input block? Let's maybe omit this.